.happy/terraform/modules/sfn_config/nextstrain.wdl

```diff
@@ -65,6 +68,7 @@ task nextstrain_workflow {
                       --all-group-sequences                                                                                      \
                       --builds-template-file /usr/src/app/aspen/workflows/nextstrain_run/builds_templates/~{template_filename}   \
                       --builds-template-args '~{template_args}'
+                      --tree-type "~{tree_type}"
```

Comment:
does nextstrain-ondemand.wdl need to be modified as well?

Reply:
I think I answered my own question - this step is handled by our api endpoint for ondemand builds so we don't need to modify the other wdl.